code/modules/blob2/overmind/types/barnacle.dm

```diff
@@ -0,0 +1,40 @@
+
+/datum/blob_type/barnacle
+	name = "barnacloid blob"
+	desc = "A mass that seems densely wound around its core."
+	ai_desc = "sedentary"
+	effect_desc = "Will not create any nodes. Has slightly higher resistances."
+	difficulty = BLOB_DIFFICULTY_EASY
+	color = "#383838"
+	complementary_color = "#c04761"
+	can_build_nodes = FALSE
+	spread_modifier = 0.1
+	ai_aggressiveness = 0
+
+	brute_multiplier = 0.35
+	burn_multiplier = 0.8
+
+/datum/blob_type/barnacle/on_chunk_use(obj/item/weapon/blobcore_chunk/B, mob/living/user)
+	var/turf/T = get_turf(B)
+
+	to_chat(user, "<span class='alien'>\The [B] produces a cauterizing ooze!</span>")
+
+	T.visible_message("<span class='alium'>\The [B] shudders at \the [user]'s touch, before disgorging a disgusting ooze.</span>")
```

Comment:
Should you include `usser` in the exclude_mobs list for this call, given the `to_chat()` to them above?
```
/atom/proc/visible_message(var/message, var/blind_message, var/list/exclude_mobs, [...]
```